src/content/en/2022/javascript.md

```diff
@@ -6,7 +6,7 @@ description: JavaScript chapter of the 2022 Web Almanac covering the usage of Ja
 authors: [malchata, ibnesayeed]
 reviewers: [mgechev, pankajparkar, NishuGoel, seldo, housseindjirdeh, kevinfarrugia]
 analysts: [NishuGoel, kevinfarrugia]
```

Comment:
@malchata can you suggest edits to these lists to ensure that they only contain people who actually contributed and are roughly ordered so the ones with the biggest contributions are first?

Reply:
Did quite a thorough review here, so going to stick myself down in the reviewer section:

```suggestion
authors: [malchata, ibnesayeed]
reviewers: [mgechev, pankajparkar, NishuGoel, seldo, housseindjirdeh, kevinfarrugia, tunetheweb]
analysts: [NishuGoel, kevinfarrugia]
```